setup.py

```diff
@@ -2003,16 +2003,9 @@ def detect_ctypes(self, inc_dirs, lib_dirs):
             ffi_inc = find_file('ffi.h', [], inc_dirs)
         if ffi_inc is not None:
```

Comment:
@zware: the problem is that find_file does not always get called and moreover that message may be useful to pinpoint a build problem,